server/socket/socketHelperFunctions.js

```diff
@@ -105,6 +105,30 @@ exports.updateRobotJobStatus = async (jobId, status) => {
   });
 };
 
+/**
+ * @description Finds a specific job in the database and updates the status of the Job
```

Comment:
The naming of the function and the documentation above it does not really seem to match up. On one side the talk is about errors, on the other side it is status in general